nav2_behavior_tree/nav2_tree_nodes.xml

```diff
@@ -14,6 +14,7 @@
       <input_port name="server_name">Server name</input_port>
       <input_port name="server_timeout">Server timeout</input_port>
       <output_port name="error_code_id">"Back up error code"</output_port>
+      <output_port name="error_msg">"Back up error message"</output_port>
```

Comment:
Ditto: check here to make sure any updates made are reflected as a final verification